packages/perseus-editor/src/components/__stories__/locked-point-settings.stories.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from "react";
+
+import LockedPointSettings from "../locked-point-settings";
+import {getDefaultFigureForType} from "../util";
+
+import type {Meta, StoryObj} from "@storybook/react";
+
+export default {
+    title: "Perseus Editor/Components/Locked Point Settings",
+    component: LockedPointSettings,
+} as Meta<typeof LockedPointSettings>;
+
+export const Default = (args): React.ReactElement => {
+    return <LockedPointSettings {...args} />;
+};
+
+type StoryComponentType = StoryObj<typeof LockedPointSettings>;
+
+// Set the default values in the control panel.
+Default.args = {
+    ...getDefaultFigureForType("point"),
+    onChangeProps: () => {},
+};
+
+export const Controlled: StoryComponentType = {
+    render: function Render() {
+        const [props, setProps] = React.useState(
+            getDefaultFigureForType("point"),
+        );
+
+        const handlePropsUpdate = (newProps) => {
+            setProps({
+                ...props,
+                ...newProps,
+            });
+        };
+
+        return (
+            <LockedPointSettings {...props} onChangeProps={handlePropsUpdate} />
+        );
+    },
+};
+
+/**
+ * In some cases, the locked point may be shown or hidden from the graph,
+ * such as when the point is used to define a locked line.
+ *
+ * Use `onToggle` and `toggled` to update the visibility of the locked point.
+ * When `onToggled` is passed in, a switch will be rendered to allow
+ * toggling the point, which will then show/hide the relevant properties
+ * for the locked point.
+ */
```

Comment:
**thought**: I wonder if this would be better as rendered text in the story? Then engineers viewing this story would see this text in Storybook. What do you think?